rust-rustfmt.el

```diff
@@ -33,7 +33,7 @@
   :type 'string
   :group 'rust-mode)
 
-(defcustom rust-rustfmt-switches '("--edition" "2024")
+(defcustom rust-rustfmt-switches nil
   "Arguments to pass when invoking the `rustfmt' executable."
```

Comment:
It may be helpful to add some documentation here.  Feel free to reword it, but I was thinking of something like "It is recommended that rustfmt be configured by a per-user rustfmt.toml rather than this variable, because command-line switches specified here will override per-project settings."

Reply:
Good point, I would probably reword it in this way:
> Arguments to pass when invoking the `rustfmt' executable. This variable will override any user configuration (e.g. rustfmt.toml)

Reply:
Looks good, I will probably add something like to make sure we convey it further:

> Arguments to pass when invoking the `rustfmt' executable. This variable will override any user configuration (e.g. rustfmt.toml). Recommendation is to not modify this and rely on declarative configuration instead.

Basically the intention is to convey the new users to not use the option unless they know what they are doing.